api/query/cache/service/main.go

```diff
@@ -34,6 +35,39 @@ var (
 
 	idx index.Index
 	mon monitor.Monitor
+
+	// Base query to AND against user's query: only include (sub)tests with
+	// non-UNKNOWN status for at least one of the four major browsers.
+	// I.e.,
+	//     !chrome:UNKNOWN | !edge:UNKOWN | !firefox:UNKNOWN | !safari:UNKNOWN
+	baseQuery = query.AbstractOr{
```

Comment:
This is tightly coupled to the default browser set, which 
a) doesn't handle say, uc-browser
b) has more non-empty checks than it needs to on say, chrome v chrome

Can we make this a function of the runs that are being queried, or can we implement an `!any-browser:UNKNOWN` type of thing?

Reply:
@lukebjerring Just updated code to construct a concrete query over actual run IDs from the request. PTAL.